src/routes/embedded/account/backup-full-wallet/backup-full-wallet.view.tsx

```diff
@@ -0,0 +1,84 @@
+import { FolderShield, Key01 } from "@untitled-ui/icons-react";
+import copy from "copy-to-clipboard";
+import { useState, useEffect } from "react";
+import {
+  Box,
+  Button,
+  Card,
+  WanderFooter,
+  Copyable,
+  WarningIcon,
+  Snackbar
+} from "~components/embed/ui";
+import { useEmbedded } from "~utils/embedded/embedded.hooks";
+import { WalletUtils } from "~utils/wallets/wallets.utils";
+import { useLocation } from "~wallets/router/router.utils";
+
+export function AccountBackupFullWalletEmbeddedView() {
+  const { navigate } = useLocation();
+  const { currentWallet, downloadKeyfile } = useEmbedded();
+
+  const [hasEncryptedSeedPhrase, setHasEncryptedSeedPhrase] = useState(false);
+
+  useEffect(() => {
+    WalletUtils.hasEncryptedSeedPhrase(currentWallet.id).then(
+      (hasEncryptedSeedPhrase) => {
+        setHasEncryptedSeedPhrase(hasEncryptedSeedPhrase);
+      }
+    );
+  }, [currentWallet.id]);
+
+  return (
+    <Card
+      headerText="Backup your full wallet"
```

Comment:
Did they write this like this on Figma? "backup" instead of "export"?

Reply:
Yes